build.psm1

```diff
@@ -1858,16 +1858,25 @@ function Publish-NuGetFeed
 }
 
 function Start-DevPowerShell {
+    [CmdletBinding(DefaultParameterSetName='ConfigurationParamSet')]
     param(
-        [string[]]$ArgumentList = '',
+        [string[]]$ArgumentList = @(),
         [switch]$LoadProfile,
-        [string]$binDir = (Split-Path (New-PSOptions).Output),
+        [Parameter(ParameterSetName='ConfigurationParamSet')]
+        [ValidateSet("Debug", "Release", "CodeCoverage", '')] # should match New-PSOptions -Configuration values
+        [string]$Configuration,
+        [Parameter(ParameterSetName='BinDirParamSet')]
+        [string]$binDir,
```

Comment:
I believe we follow Pascal casing for Parameters . Requesting you to change `$binDir` to `$BinDir`